troposphere/__init__.py

```diff
@@ -580,7 +580,7 @@ class Parameter(AWSDeclaration):
     NUMBER_PROPERTIES = ['MaxValue', 'MinValue']
     props = {
         'Type': (basestring, True),
-        'Default': (basestring, False),
+        'Default': (validators.string_or_integer, False),
```

Comment:
Thank you for the PR. Actually a new validator isn't needed. Using validators.integer will allow for both a string and integer value to be validated.